numpydoc/docscrape.py

```diff
@@ -500,8 +509,8 @@ def __init__(self, cls, doc=None, modulename='', func_doc=FunctionDoc,
             raise ValueError("Expected a class or None, but got %r" % cls)
         self._cls = cls
 
-        self.show_inherited_members = config.get('show_inherited_class_members',
-                                                 True)
+        self.show_inherited_members = \
```

Comment:
no backslashes please, that's bad style. if this is copied from scikit-learn, it should be changed there. If you wanted to get this line to <80 chars, a better way would be

    self.show_inherited_members = config.get(
                'show_inherited_class_members', True)